_releases/ocp-4.16-test-jobs-amd64.json

```diff
@@ -1,5 +1,22 @@
 {
   "nightly": [
+    {
+      "prowJob": "periodic-ci-openshift-openshift-tests-private-release-4.16-automated-release-aws-ipi-f999"
```

Comment:
@rioliu-rh should we add these configurations or enable the disabled ones?

Reply:
we'd better to do clean up old jobs, then add new jobs here.